Libraries/rocSPARSE/level_2/spitsv/README.md

```diff
@@ -0,0 +1,124 @@
+# rocSPARSE Level 2 Triangular Solver Example
+## Description
+This example illustrates the use of the `rocSPARSE` level 2 iterative triangular solver with a CSR sparse format.
+
+This triangular solver is used to solve a linear system of the form
+
+$$
+A'y = \alpha x,
```

Comment:
Can we use op(A) rather than A' ? This is confusing.

Reply:
Hmm, this is also present in many other examples from most other libraries, I propose to update either all of them or none, to keep the notation consistent. If this notation is preferred, I can create an internal issue and we'll upstream the fix when it's done (so in another PR)

Reply:
I would suggest to update all of them. Feel free to do it in a separate PR, but A' is not a good notation.

Reply:
Noted 👍🏼. I'll open a separate PR when this is done